server/lib/template/helpers/logic.js

```diff
@@ -23,6 +22,11 @@ function lt(a, b, options) {
   return options.inverse(this);
 }
 
+function setdefault(val, _default) {
```

Comment:
Could you rename this function to something like `ignoreNan` or `hideNan`, and also instead of adding ' ' in the handlebars you could just add it from this function like : 
```js
function ignoreNan(val) {
  return (val >= 0 || val < 0) ? val : '';
}
```